Promote VB What's new

rpetrusha opened the pull request:
## Promoted VB What's new

This PR:

- Moves the VB What's new topic up one level in the TOC to make it more discoverable.
- Corrects the casing and formatting of some headers and list items.
- Notes which versions of Visual Basic were released with which versions of Visual Studio.

Fixes #6772 

//cc @KathleenDollard

Commit:
Promoted VB What's new

Commit:
Changed formatting

KathleenDollard commented:
👍

JRAlexander commented on docs/visual-basic/getting-started/whats-new.md:
```diff
@@ -153,16 +153,16 @@ You can test for null in a very light syntactic way before performing a member a
+You can put comments after implicit line continuations, inside initializer expressions, and among LINQ expression terms.  
   
- Smarter Fully-qualified Name Resolution  
+**Smarter Fully-qualified Name Resolution**  
```
Headings should be sentence-case.

JRAlexander commented on docs/visual-basic/getting-started/whats-new.md:
```diff
@@ -153,16 +153,16 @@ You can test for null in a very light syntactic way before performing a member a
  Given code such as `Threading.Thread.Sleep(1000)`, Visual Basic used to look up the namespace "Threading", discover it was ambiguous between System.Threading and System.Windows.Threading, and then report an error.  Visual Basic now considers both possible namespaces together.  If you show the completion list, the Visual Studio editor lists members from both types in the completion list.  
   
- Year-first Date Literals  
+ **Year-first Date Literals**  
```
Heading.

JRAlexander commented on docs/visual-basic/getting-started/whats-new.md:
```diff
@@ -153,16 +153,16 @@ You can test for null in a very light syntactic way before performing a member a
  You can have date literals in yyyy-mm-dd format, `#2015-03-17 16:10 PM#`.  
   
- Readonly Interface Properties  
+ **Readonly Interface Properties**  
```
Heading here.

JRAlexander commented on docs/visual-basic/getting-started/whats-new.md:
```diff
@@ -171,7 +171,7 @@ You can put comments after implicit line continuations, inside initializer expre
  You can disable and enable specific warnings for regions within a source file.  
   
- XML Doc-comment Improvements  
+ **XML Doc-comment Improvements**  
```
Heading here.

JRAlexander commented on docs/visual-basic/getting-started/whats-new.md:
```diff
@@ -183,10 +183,10 @@ You can put comments after implicit line continuations, inside initializer expre
  If you add the `Overrides` modifier to a definition, the compiler implicitly adds `Overloads` so that you can type less code in common cases.  
   
- CObj Allowed in Attributes Arguments  
+ **CObj Allowed in Attributes Arguments**  
```
Heading here

JRAlexander commented on docs/visual-basic/getting-started/whats-new.md:
```diff
@@ -183,10 +183,10 @@ You can put comments after implicit line continuations, inside initializer expre
  The compiler used to give an error that CObj(…) was not a constant when used in attribute constructions.  
   
- Declaring and Consuming Ambiguous Methods from Different Interfaces  
+ **Declaring and Consuming Ambiguous Methods from Different Interfaces**  
```
Heading here.

JRAlexander reviewed (changes requested):
Looks good, @rpetrusha, except for the headings. They should only have the first word capitalized.

Commit:
Changed case of headers, added Visual Studio versions

rpetrusha commented:
I've addressed your comments, @JRAlexander, if you'd like to take a look again.
@davkean, I've added Visual Studio versions for the last Visual Basic releases.
@KathleenDollard, you might want to take a look at the latest changes.

JRAlexander commented:
LGTM, @rpetrusha!
